discord/ext/ipcx/server.py

```diff
@@ -99,6 +102,8 @@ def __init__(
         self.do_multicast = do_multicast
         self.multicast_port = multicast_port
 
+        self.max_msg_size = max_msg_size
```

Comment:
There isn't a need for the user to handle this. Besides, the code here passes `None` by default, which is unnecessary as this implies that there is an default number

Reply:
i removed the max_msg_size parameter from Server and simply set it to 0, which removes the limit